packages/lesswrong/components/tagging/TagPage.tsx

```diff
@@ -25,36 +22,23 @@ const styles = theme => ({
 const TagPage = ({classes}: {
   classes: ClassesType
 }) => {
-  const { SingleColumnSection, SectionTitle, SectionFooter, SectionButton, PostsItem2, ContentItemBody, Loading, Error404, LoadMore } = Components;
+  const { SingleColumnSection, SectionTitle, PostsList2, SectionButton, ContentItemBody, Loading, Error404 } = Components;
   const currentUser = useCurrentUser();
-  const { params } = useLocation();
-  const { slug } = params;
+  const { query, params: { slug } } = useLocation();
   const { tag, loading: loadingTag } = useTagBySlug(slug);
 
-  const { results, loading: loadingPosts, loadMoreProps } = useMulti({
-    skip: !(tag?._id),
-    terms: {
-      view: "postsWithTag",
-      tagId: tag?._id,
-    },
-    collection: TagRels,
-    fragmentName: "TagRelFragment",
-    limit: 300,
-    itemsPerPage: 100,
-    enableTotal: true,
-    ssr: true,
-  });
-
-
   if (loadingTag)
     return <Loading/>
   if (!tag)
     return <Error404/>
 
-  // Filter out all tagRels where we don't have access to the corresponding post
-  const filteredResults = _.filter(results, result => !!result.post)
-  const orderByScore = _.sortBy(filteredResults, result=>-result.post.baseScore)
-  const orderByTagScore = _.sortBy(orderByScore, result=>-result.baseScore)
+  const terms = {
+    ...query,
+    filterSettings: {tags:[{tagId: "tNsqhzTibgGJKPEWB", tagName: "Coronavirus", filterMode: "Required"}]},
```

Comment:
This looks like debug stuff left behind? The tag page should not have a hardcoded tag ID.

Reply:
Ah, whoops. Fixed.